example_app_frames/src/main/java/com/checkout/example/frames/ui/screen/MainActivity.kt

```diff
@@ -37,6 +47,36 @@ fun Navigator(
     onSuccess: (TokenDetails) -> Unit,
     onFailure: (String) -> Unit
 ) {
+    val paymentFormThemeColors = PaymentFormThemeColors(
+        paymentFormColors = PaymentFormColors(background = 0xFF17201E),
+        buttonColors = PaymentFormButtonColors(
+            containerColor = 0XFF00CC2D,
```

Comment:
create a local var to for all hex colors and re-use it elsewhere in the theming for readability. For example 0XFF00CC2D is used multiple times.